dev/com.ibm.ws.security.acme/src/com/ibm/ws/security/acme/internal/AcmeProviderServiceImpl.java

```diff
@@ -38,73 +37,86 @@
 public class AcmeProviderServiceImpl implements AcmeConfig, ServletContextListener, ServletContainerInitializer {
 
     private final TraceComponent tc = Tr.register(AcmeProviderServiceImpl.class);
-    private final String parameter1 = "JM-acme-parm1";
-    private final String parameter2 = "JM-acme-parm2";
 
     private AcmeService serviceProvider;
-
-    /**
-     * The properties class that contain the attributes defined
-     * by inside of server.xml.
-     */
-    private final Properties sessionProperties = new Properties();
-    /**
-     * Strings used to access the various attributes that are
-     * defined in the <mailSession> and that are subsequently
-     * extracted from the ComponentContext to be placed in the Properties
-     * object
-     */
-    public static final String PARM1 = "configParm1";
-    public static final String PARM2 = "configParm2";
-
-    private final String propertiesArray[] = { PARM1, PARM2 };
+    
+	private String directoryURI;
+	private String domain;
+	private int validFor;
+	private String country;
+	private String locality;
+	private String state;
+	private String organization;
+	private String organizationalUnit;
+
+	// Challenge and order related fields.
+	private int challengeRetries;
+	private int challengeRetryWait;
+	private int orderRetries;
+	private int orderRetryWait;
+
+	// ACME account related fields.
+	private String accountKeyFile;
+	private Set<String> accountContact;
+	private boolean acceptTermsOfService;
+	private String domainKeyFile;
+
+	private AcmeClient acmeClient;
 
     private final HashMap<String, Set<String>> appModules = new HashMap<String, Set<String>>();
 
-    // File name of the User Key Pair
-    private static final File USER_KEY_FILE = new File("user.key");
-
-    // File name of the Domain Key Pair
-    private static final File DOMAIN_KEY_FILE = new File("domain.key");
-
-    // File name of the CSR
-    private static final File DOMAIN_CSR_FILE = new File("domain.csr");
-
-    // File name of the signed certificate
-    private static final File DOMAIN_CHAIN_FILE = new File("domain-chain.crt");
-
-    //Challenge type to be used
-    private static final ChallengeType CHALLENGE_TYPE = ChallengeType.HTTP;
-
-    // RSA key size of generated key pairs
-    private static final int KEY_SIZE = 2048;
-
-    private enum ChallengeType {
-        HTTP, DNS
-    }
-
     @Activate
     public void activate(ComponentContext context, Map<String, Object> properties) {
-        Tr.info(tc, "******* JTM ******* AcmeProviderServiceImpl: inside activate() method. Display input properties: ");
-
+		initialize(properties);
     }
 
     @Modified
     public void modify(Map<String, Object> properties) {
-        Tr.info(tc, " ******* JTM ******* AcmeProviderServiceImpl: inside modified () method");
+		initialize(properties);
     }
 
+	public void initialize(Map<String, Object> configProps) {
+		Tr.info(tc, "Initialize ACME");
```

Comment:
Probably shouldn't be an 'info'.

Reply:
Removed or commented out all the info's in this class.